components/calendar/src/lib.rs

```diff
@@ -167,10 +170,10 @@ pub mod japanese;
 pub mod julian;
 pub mod provider;
 pub mod types;
+pub mod week_of;
 
 pub use any_calendar::{AnyCalendar, AnyCalendarKind};
 pub use calendar::Calendar;
-pub use calendar_arithmetic::{ArithmeticDate, CalendarArithmetic};
 pub use date::{AsCalendar, Date, Ref};
```

Comment:
`Ref` has its whole doc in one paragraph which takes up a lot of space in the docs:
<img width="899" alt="image" src="https://user-images.githubusercontent.com/4706271/185613821-9c24583e-0b7d-4f2b-9657-f96098f255d1.png">